src/NuGet.Clients/NuGet.PackageManagement.UI/Models/PackageDetailControlModel.cs

```diff
@@ -27,6 +29,8 @@ public class PackageDetailControlModel : DetailControlModel
         {
             _solutionManager = solutionManager;
             _solutionManager.ProjectUpdated += ProjectChanged;
+
+            VersionsView = new CollectionViewSource() { Source = Versions }.View;
```

Comment:
Is this necessary at this point? Can we have versions without calling `CreateVersionsAsync` ?